notebooks/279-photo-maker/279-photo-maker.ipynb

```diff
@@ -0,0 +1,1029 @@
+{
```

Comment:
<p>removing torch.jit cache does not lead to removing pytorch model outside convert function. I think you can remove them explicitly and collect memory using garbage collector</p><p> </p><p>(gc.collect() after model converted and does not used anymore)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
done